packages/snip20/Readme.md

```diff
@@ -0,0 +1,58 @@
+# Secret Contract Development Toolkit - SNIP20 Interface
+
+These functions are meant to help you easily interact with SNIP20 compliant tokens.  
+
+## Handle Messages
+
+You can create a Snip20HandleMsg variant and call the `to_cosmos_msg` function to generate the CosmosMsg that shoud be pushed onto the InitResponse or HandleResponse `messages` Vec.<br/>
+Or you can call the individual function for each Handle message.<br/>
+You probably have also noticed that CreateViewingKey is not supported.  This is because a contract can not see the viewing key that is returned because it has already finished executing by the time CreateViewingKey would be called.  If a contract needs to have a viewing key, it must create its own sufficiently complex viewing key, and pass it as a parameter to SetViewingKey. You can see an example of creating a complex viewing key in the [Snip20 Reference Implementation](http://github.com/enigmampc/snip20-reference-impl).  It is also highly recommended that you use the block_size padding option to also mask the length of the viewing key your contract has generated.
```

Comment:
you don't need the `<br>`s, you can just put two newlines between the paragraphs and it'll be split.